tutorials/concepts/arch.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Landscape Architecture"
+weight = 4
+template = "doc.html"
++++
+
+# Landscape Architecture
+
+Landscape is designed using a particular userspace architecture model known as store/hook/thread. This model divides applications into multiple parts, two gall applications and threads.
+
+## Stores
+
+Stores are the first of the two gall models to form this architecture. A store is intended to maintain all the state required for an application or set of applications. Stores are intended to be only accessible by the ship it's running on. While this is not enforced, it is the convention. The primary example of a store is `graph-store` which can be found at `app/graph-store`
```

Comment:
Add:

“A store may not send pokes, peeks, or watches. A store should be treated and thought of as a database table.”